_locales/en/messages.json

```diff
@@ -135,6 +135,10 @@
     "message": "Double-click to maximize/restore the height",
     "description": "Tooltip for the resize grip in style editor"
   },
+  "cm_linter": {
```

Comment:
Now cm_linter causes a conflict with master. Better move it to maintain sorting order within cm_* messages. 

We still have some messages that were added at random positions, but I fix those when doing something meaningful with the related code.

Reply:
My OCD kicked in and I ended up sorting all the "cm_*" messages... I reined it in before sorting all of the entries.

Reply:
Ahh derp.. you mean conflicts with the master branch... I need more caffeine.